### PR TITLE
#164238429 Fix Sign Out button

### PR DESCRIPTION
**Pull request by asamisellah**
#### What does this PR do?
Fixes sign out button by closing it when out of focus and aligns the dropdown icon on the filter buttons to the text

#### Description of Task to be completed?
- align dropdown icon in filter button with the text
- close sign out button when out of focus

#### How should this be manually tested?
1. Open the application on your browser
2. Sign in
3. On the header, click on the dropdown button on the far right
4. A sign out button should appear
5. Click away from the button
6. The sign out button should close/ disappear
7. Check that the dropdown icons on the filter buttons is well aligned with the button text
8. Check the filter and search buttons to ensure that they are interactive
9. Run `npm test` to ensure that the tests pass

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[#164238429](https://www.pivotaltracker.com/n/projects/2197045/stories/164238429)

### CODE REVIEW

**Comment by valmayaki** [comment]
Looked at this pr and its working as it should but what remains is the codeclimate issue.

File: src/__tests__/components/Header.test.jsx [hunk 1]
```diff
@@ -62,4 +63,12 @@ describe('toggleSignoutDropDown() method', () => {
     expect(renderedComponent.toggleSignoutDropDown.calledOnce).toEqual(true);
     expect(getComponent().state().signoutDropDownIsVisible).toEqual(true);
   });
+
+  it('sign out button should close on clicking away', () => {
+    // Find the element then click
+    // click on the button(display is true)
+    // click away(click anywhere in the dom)
```
Comment (davenjeru) [inline src/__tests__/components/Header.test.jsx]: Are these the tests that should bring the coverage back up? @asamisellah

Comment (dotNesh) [reply src/__tests__/components/Header.test.jsx]: No. For test coverage, the thought doesn't count. 😂 @asamisellah

**Comment by valmayaki** [comment]
it seems code climate is angry more methods no test covering them :(

File: src/__tests__/components/Header.test.jsx (hunk 1 above)
Comment (asamisellah) [reply src/__tests__/components/Header.test.jsx]: Hahaha... I'm working on increasing the test coverage. was making some changes to the filter button. Kindly have a look at that as well. Thanks in advance

**Pull request merged**